public/robots.txt

```diff
@@ -0,0 +1,9 @@
+# *
```

Comment:
Delete this file locally. It's no longer required and has been removed from the gitignore.